src/__init__.py

```diff
@@ -30,46 +24,12 @@
 app.config['SQLALCHEMY_ECHO'] = False  # for print the sql query
 
 db = SQLAlchemy(app)
-
-# Other Services
-from src.services.mqtt_client.mqtt_client import MqttClient
-from src.services.histories.history_local import HistoryLocal
-from src.services.histories.point_store_history_cleaner import PointStoreHistoryCleaner
-# Source Drivers
-# from src.source_drivers.bacnet.services.network import Network
-from src.source_drivers.modbus.services.tcp_registry import TcpRegistry
-from src.source_drivers.modbus.services.tcp_polling import TcpPolling
-from src.source_drivers.modbus.services.rtu_polling import RtuPolling
-from src.source_drivers.modbus.services.rtu_registry import RtuRegistry
-
-from src import routes
+from src import routes  # importing for creating all the schema on un-existing case
 
 db.create_all()
```

Comment:
check this fixes #74

Reply:
`point_store_history` were not creating on db, which is also handled.